stdlib/crypto/src/main/ballerina/src/crypto/Module.md

```diff
@@ -1,5 +1,3 @@
-## Module overview
-
 This module provides the necessary utilities that are required to hash content using different hashing mechanisms and algorithms. 
 
 ## Samples
```

Comment:
I think we need remove this section and point it to the BBE as per the guideline.

Reply:
Removed and pointed to the BBE.